src/sql/workbench/contrib/notebook/browser/notebookViews/notebookViewsGrid.css

```diff
@@ -2,127 +2,445 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
+:root .grid-stack-item > .ui-resizable-handle {
+	filter: none;
+}
+.grid-stack {
```

Comment:
I would suggest removing the grid-stack class. You already have nb-grid-stack, and grid-stack itself is too generic currently. 

(if we wanted to make this a generic thing other features could use then we could consider that. But for now this seems to be extremely specific to notebooks so it'd be better to keep this clearly tied to them)

Reply:
Sounds good